extra/resources/ifspeed

```diff
@@ -480,4 +529,4 @@ case $__OCF_ACTION in
         ;;
 esac
 
-exit $?
+exit $?
```

Comment:
This (deleted new-line character) is probably overlooked. Could you please return it back?